### PR TITLE
Better default ReadMe for ROS 2 repositories.

### PR DESCRIPTION
**Pull request by kscottz**
Strawman proposal for a more comprehensive README.md for most of the core ROS 2 repositories.  My goal with this readme is to quickly summarize most of the core community resources for new ROS users. 

Once we converge on a final proposal my intent is to re-use this text at the end of most of the ROS 2 repos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+# Join the [ROS](https://ros.org/) Community
+
+## Community Resources
+
+* [ROS Discussion Forum](https://discourse.ros.org/)
+* [ROS Discord Server](https://discord.com/servers/open-robotics-1077825543698927656)
+* [Robotics Stack Exchange](https://robotics.stackexchange.com/) (preferred ROS support forum).
+* [Official ROS Videos](https://vimeo.com/osrfoundation)
+* [ROSCon,](https://roscon.ros.org) our yearly developer conference.
```
Comment (clalancette) [inline README.md:15]:
```suggestion
* [ROSCon](https://roscon.ros.org), our yearly developer conference. 
```

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+* [ROS 1 Wiki](https://wiki.ros.org/)
+
+## Developer Resources
+* [ROS 2 Documentation](https://docs.ros.org/)
+* [ROS Package Python APIs](https://docs.ros.org/en/rolling/p/)
+* [ROS Package C++ APIs](https://docs.ros.org/en/api/)
+* [ROS Package Index](https://index.ros.org/)
+* [ROS on Docker Hub](https://hub.docker.com/_/ros/)
+* [ROS Resource Status Page](https://status.openrobotics.org/)
+* [REP-2000:](https://ros.org/reps/rep-2000.html) ROS 2 Releases and Target Platforms
```
Comment (clalancette) [inline README.md:26]:
```suggestion
* [REP-2000](https://ros.org/reps/rep-2000.html): ROS 2 Releases and Target Platforms
```

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+
+## Community Resources
+
+* [ROS Discussion Forum](https://discourse.ros.org/)
+* [ROS Discord Server](https://discord.com/servers/open-robotics-1077825543698927656)
+* [Robotics Stack Exchange](https://robotics.stackexchange.com/) (preferred ROS support forum).
+* [Official ROS Videos](https://vimeo.com/osrfoundation)
+* [ROSCon,](https://roscon.ros.org) our yearly developer conference. 
+* Cite ROS 2 in academic work using [DOI: 10.1126/scirobotics.abm6074](https://www.science.org/doi/10.1126/scirobotics.abm6074) 
+* [ROS 1 Wiki](https://wiki.ros.org/)
```
Comment (clalancette) [inline README.md:17]: I don't think we should reference the ROS 1 wiki here; that is going to eventually go away.

```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1 +1,37 @@
...
+
+## Developer Resources
+* [ROS 2 Documentation](https://docs.ros.org/)
+* [ROS Package Python APIs](https://docs.ros.org/en/rolling/p/)
+* [ROS Package C++ APIs](https://docs.ros.org/en/api/)
```
Comment (clalancette) [inline README.md:22]:
```suggestion
* [ROS Package API reference](https://docs.ros.org/en/rolling/p/)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,37 @@
...
+## Project Resources
+* [Purchase ROS Swag](https://spring.ros.org/)
+* [Information about the ROS Trademark](https://www.ros.org/blog/media/)
+* On Social Media
+  * [Open Robotics on LinkedIn](https://www.linkedin.com/company/open-source-robotics-foundation)
+  * [Open Robotics on Twitter](https://twitter.com/OpenRoboticsOrg)
+  * [ROS.org on Twitter](https://twitter.com/ROSOrg)
+
+ROS is made possible through the generous support of open source contributors and the non-profit [Open Source Robotics Foundation (OSRF)](https://www.openrobotics.org/). Tax deductible donations to the OSRF can be [made here.](https://donorbox.org/support-open-robotics?utm_medium=qrcode&utm_source=qrcode)
+
```
Comment (clalancette) [inline README.md:37]: Remove trailing whitespace.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kscottz) [reply README.md:17]: I am always on the fence on this one but I'll agree in this case.

File: README.md (hunk 5 above)
Comment (kscottz) [reply README.md]: You got it.

**Review by fujitatomoya — APPROVED** [review]
lgtm

**Review by clalancette — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,34 @@
-Documentation is at https://docs.ros.org
+# About 
+The Robot Operating System (ROS) is a set of software libraries and tools that help you build robot applications. From drivers to state-of-the-art algorithms, and with powerful developer tools, ROS has what you need for your next robotics project. And it's all open source. Full project details on [ROS.org](https://ros.org/)
+
+# Getting Started 
+Looking to get started with ROS? Our [installation guide is here](https://www.ros.org/blog/getting-started/). Once you've installed ROS start by learning some [basic concepts](https://docs.ros.org/en/rolling/Concepts/Basic.html) and take a look at our [beginner tutorials](https://docs.ros.org/en/rolling/Tutorials/Beginner-CLI-Tools.html).
+
+# Join the [ROS](https://ros.org/) Community
```
Comment (gbiggs) [inline README.md:7]:
```suggestion
# Join the ROS Community
```

**Review by gbiggs — COMMENTED** [review]
Apart from one tiny change, I think this looks fantastic. Let's get it in all the repositories!

**Review by gbiggs — APPROVED** [review]

**Pull request merged**